src/System Application/App/File System/app.json

```diff
@@ -0,0 +1,70 @@
+{
+    "id":  "c9c54414-80c3-4cc9-98c6-589158882774",
+    "name":  "File System",
+    "publisher":  "Microsoft",
+    "brief":  "Enables access to external file systems from Business Central.",
+    "description":  "Enables access to external file systems from Business Central.",
+    "version":  "25.0.0.0",
```

Comment:
Let's get all of those versions updated to 26 and let's get this in.

Reply:
Done.